orchestrator/lib/audit/audit.go

```diff
@@ -39,6 +29,10 @@ func Event(action fhir.AuditEventAction, resourceReference *fhir.Reference, acti
 		interactionDisplay = "Search"
 	}
 
+	if localIdentity == nil {
```

Comment:
then just make it a non-pointer here, then you don't need to nil-check (`getLocalIdentity()` already returns an error, so it can do the nil check)